go.mod

```diff
@@ -137,3 +137,9 @@ require (
 	gotest.tools v2.2.0+incompatible // indirect
 	k8s.io/apimachinery v0.17.1 // indirect
 )
+
+// replaced due to https://github.com/satori/go.uuid/issues/73
+replace github.com/satori/go.uuid => github.com/gofrs/uuid v3.2.0+incompatible
+
+// Set the Azure Event Hub log abstraction to the Telegraf format:
+replace github.com/devigned/tab => github.com/R290/tab v0.1.2
```

Comment:
I'd like to skip this change, generally I don't like to allow libraries to access the Logger, in the case we did want to log the library we would hook it all up to debug level messages.  However, if we want to do that instead of forking the library we should create a custom tab.Tracer, tab.Spanner, tab.Logger, and then call tab.Register(tracer).

Reply:
Merged the pull request for removing the log abstraction replacement.